dist/formatters/no-internal-summary-table-creator.js

```diff
@@ -0,0 +1,126 @@
+const path = require('path');
+const fs = require('fs');
+
+module.exports = function(messages, ruleId) {
+  const problemFiles = new Map();
+  const errorTracker = new Map();
+  const tagViolationsTracker = new Map();
+  const currentWorkingDirectory = process.cwd();
```

Comment:
please also import process, which is a global

Reply:
done